storybook/src/stories/Tables/molecules/ActionsTable.stories.tsx

```diff
@@ -0,0 +1,215 @@
+import React, {useState, useCallback, ReactElement} from 'react';
+import styled from 'styled-components';
+import { object, boolean } from "@storybook/addon-knobs";
+import { action } from '@storybook/addon-actions';
+
+
+import {
+TypeTable as TypeTableCustom,
+ActionButtons,
+IconButtonData,
+MultilineContent,
```

Comment:
These are the two new Templates for Cell

Reply:
👍